### DISCUSSION TITLE
option to have the audio global to the room and not spatialized.

### DISCUSSION THREAD

**Discussion by salahzar** [mozilla/hubs#3019] [General Questions & Discussion] [answered]
Is there already be the possibility of having the voice (especially of the conferencer or organizer) hearable from all the room without being obliged to be very close to them?
I thought that the /audiomode would serve that task, but after testing it, it seems not working.

**Comment by johnshaughnessy** [comment, marked as answer]
> Is there already be the possibility of having the voice (especially of the conferencer or organizer) hearable from all the room without being obliged to be very close to them?

There is an [open pull request implementing a version of this feature](https://github.com/mozilla/hubs/pull/2766), but it has not been merged because we want to discuss alternative ways to achieve the same end. One concern with having it user-local preferences available without an alternative is that it means audio travel distance and behavior will vary significantly between clients, and it will be difficult for people to have a correct understanding of how far their voices carry, who is within earshot, etc.

>  I thought that the /audiomode would serve that task, but after testing it, it seems not working.

At this time, `/audiomode` _only_ toggles the _panning_ of the audio (which adjusts the sound based on whether it's coming above/below, in front/behind, or from left/right), but _distance-based attenuation_ remains.


For what it's worth, it's "loud and clear" to us based on the [several open issues about audio](https://github.com/mozilla/hubs/issues?q=is%3Aissue+is%3Aopen+audio) that people care a lot about this, and it's a high priority area that the team wants to improve.

**Comment by salahzar** [comment]
@johnshaughnessy altspacevr solved this problem having a sort of megaphone that can be enabled for specific users so that they can be heard in all the current environment. That metaphore was quite interesting and intuitive. When using altspacevr this feature was really useful to keep people concentrated on the conference.

**Reply by johnshaughnessy** [reply]
Yes that seems like a good option. It's pretty similar to https://github.com/mozilla/hubs/issues/2106 . It's linked from a comment in https://github.com/mozilla/hubs/issues/2305
